### ISSUE TITLE
Subxor: true packets do not have a sub/xor defined

### ISSUE THREAD

**Issue by cyberkitsune** [PolarisTeam/PSO2Packet#15]
Any packet with `subxor: true` should probablly have the sub and xor values defined. (Given these are vital and do not change typically between PSO2 protocol versions.

IMO If possible, tests on this repo should fail if an entry has `subxor: true` but no sub or xor defines

**Comment by Kyle873** [comment]
Definitely. It'll remind whoever can do it to actually get the sub/xor for ones that have been missed.

**Comment by VariantXYZ** [comment]
I don't think we have any packets with a subxor that aren't defined in Packets.yml. 

What we do need is a set of proper tests, I was waiting to add more packets and work out possible structural issues.
